### PR TITLE
Fix `npm test` on Windows by using OS-depend path seperators

### PR DESCRIPTION
**Pull request by sbeh**

### CODE REVIEW

File: validation/markets-validator.js [hunk 1]
```diff
@@ -464,6 +479,31 @@ function MetadataValidator(metadata) {
...
+                case 'https:': request = https; break;
+                default:
+                    this.warnings.push(new CustomIssue('Unknown protocol "' + url.protocol + '" in data source url.'));
+            }
+            if (request) {
+                url.method = 'HEAD';
+                url.timeout = 10000; // 10 seconds
+                request = request.request(url, function (response) {
+                    if (response.statusCode !== 200)
+                        asyncWarnings.push(new CustomIssue(cityName + ': HTTP response status of data source url was ' + response.statusCode + '.'));
```
Comment (torfsen) [inline validation/markets-validator.js]: Some of these lines are rather long for my taste, however I'm not sure whether we have decided on a common policy, yet.

**Comment by sbeh** [comment]
Okay, thats wired.

Commits 7745593 till 0777e22 don't belong here. Should I redo the PR?

File: validation/markets-validator.js (hunk 1 above)
Comment (johnjohndoe) [inline validation/markets-validator.js]: No we don't. But I think we can still mention it as part of the review. I think @sbeh fixed the lines in his new pull request #174.

**Comment by johnjohndoe** [comment]
I rebased and merged your commit here: ba42e58d28420af10e8e1b35d3c89e9f64b918a6.

**Pull request closed without merging**